lib/LANraragi/Model/Tankoubon.pm

```diff
@@ -27,6 +28,21 @@ sub get_tankoubon_list ( $page = 0 ) {
     # Tankoubons are represented by TANK_[timestamp] in DB. Can't wait for 2038!
     my @tanks = $redis->keys('TANK_??????????');
 
+    # This can potentially get very slow searches!!!
+    if ( length($name) ) {
```

Comment:
I wouldn't bother implementing this -- all tank names are now included in LRR_TITLES when building the stat indexes, so even in the event that we need to add a search bar to a "here are all your tanks" page, I would rather rely on the existing search infrastructure.  

(probably by just swapping the initial id list to be all the tank IDs instead of the archive/tankID mix we have for base search rn)  

I'd even argue for just doing this clientside, but it might become problematic later if users start having 1000s of tanks so ¯\\\_(ツ)_/¯

Reply:
Then we can remove it, TBH it never ended up growing in me, and clientside searches are not out of place either, I think Ichaival does something similar with file searches.